docs_src/design/legacy-requirements/device-service.md

```diff
@@ -116,6 +116,9 @@ Port | Int | Port on which to contact the core-metadata service.
 
 #### Device section
 
+!!! edgey - "EdgeX 3.0"
+    **UpdateLastConnected** is removed in EdgeX 3.0.
```

Comment:
should you also remove line 129?

Reply:
This page is for the legacy requirements. I don't think we should update it, so removing the note.

Reply:
oh, so you should not modify this page at all.
please leave all the pages under /design as is